core/src/test/scala/integration/kafka/api/SaslClientsWithInvalidCredentialsTest.scala

```diff
@@ -173,4 +223,19 @@ class SaslClientsWithInvalidCredentialsTest extends IntegrationTestHarness with
       }
     }, s"Operation did not succeed within timeout after $attempts")
   }
+
+  private def createTransactionalProducer(): KafkaProducer[Array[Byte], Array[Byte]] = {
+    producerConfig.setProperty(ProducerConfig.TRANSACTIONAL_ID_CONFIG, "txclient-1")
+    producerConfig.put(ProducerConfig.MAX_IN_FLIGHT_REQUESTS_PER_CONNECTION, "5")
+    producerConfig.put(ProducerConfig.ENABLE_IDEMPOTENCE_CONFIG, "true")
+    producerConfig.put(ProducerConfig.BATCH_SIZE_CONFIG, "16384")
```

Comment:
This and max in flight are defaults right?

Reply:
yes, removed.